easybuild/easyconfigs/m/matplotlib/matplotlib-1.5.1-goolf-1.7.20-Python-2.7.11.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '1.5.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('freetype', '2.5.5'),
+    ('libpng', '1.6.18'),
```

Comment:
I have no reason, those are just the ones I use in my cluster. Should I upgrade to freetype/2.6.2 and libpng/1.6.21? or which versions should I use?